packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -158,6 +160,14 @@ governing permissions and limitations under the License.
   /* allow focus ring of child to extend outside the bounds */
   padding: 4px;
   margin: -4px;
+
+  /*
+   * This is so user provided content follows the column alignment setting.
+   * Has a specificity of 0 1 0, which is the same as a single class.
+   */
+  > * {
+    display: inline;
+  }
```

Comment:
Combines with the text-aligns added in the table.css file so that table cell content is properly aligned. 
Open question: Why do we have a RSP table.css file separate from this file?